src/resources/nfts/simplehash/utils.ts

```diff
@@ -249,18 +255,19 @@ function handleImages(
   lowResPngUrl: string | undefined;
   fullResUrl: string | undefined;
 } {
-  if (!original && !preview) {
+  if (typeof original !== 'string' || typeof preview !== 'string') {
```

Comment:
Just narrows the types to avoid coalescing

Reply:
i dont understand what this does. also did you mean to change the `&&` to `||`? if one of these conditions is true, we can still return at least some images

Reply:
is this just removing the case that the image url is `''`? i don't see why that's a bad thing